src/components/FormSelectList.vue

```diff
@@ -597,6 +636,22 @@ export default {
           }
         });
       }
+
+      if (!parsedOption.hasOwnProperty(this.optionsAriaLabel)) {
+        Object.defineProperty(parsedOption, this.optionsAriaLabel, {
```

Comment:
if will be posible we can abstract this fragment of code and promote to an reusable method